Added new template to setup a Jenkins SSH slave.

fhossfel opened the pull request:
I have added a new template to add an SSH Slave for Jenkins.

The upstream project is here https://hub.docker.com/r/jenkinsci/ssh-slave/ and https://github.com/jenkinsci/docker-ssh-slave.

The general documentation on how to setup an SSH slave in Jenkins can be found here: https://wiki.jenkins.io/display/JENKINS/SSH+Slaves+plugin

In case you have any questions please let me know.

rawmind0 commented on templates/jenkins-ssh-slave/config.yml:
```diff
@@ -0,0 +1,5 @@
+name: Jenkins SSH Slave
+description: |
+  Add additional SSH slaves to enhance a Jenkins CI server.
+version: 0.01
```
Build is failing due to error on `version` field. It should match to rancher-compose version.
```
-  version: 0.01
+  version: 0.0.1
```

rawmind0 commented on templates/jenkins-ssh-slave/docker-compose.yml:
```diff
@@ -0,0 +1,27 @@
+      GITLAB_OMNIBUS_CONFIG: |
+        external_url '${gitlab_omnipus_prefix}${gitlab_hostname}'
+        registry_external_url '${gitlab_omnipus_prefix}${registry_gitlab_hostname}'
+        gitlab_rails['gitlab_shell_ssh_port'] = ${ssh_port}
```
What is this file?? Should be out of the PR.

rawmind0 commented on templates/jenkins-ssh-slave/rancher-compose.yml:
```diff
@@ -0,0 +1,69 @@
+    unhealthy_threshold: 3
+    strategy: recreate
+    response_timeout: 3000
+    healthy_threshold: 2
```
What is this file?? Should be out of the PR.

rawmind0 commented:
Hi @fhossfel .. Thanks for the PR..

Please, take a look to changes requested.

fhossfel commented:
@rawmind0: Thanks for pointing out the incorrect version. I have updated the config.yml accordingly.

The two files were just some files I used as templates while creating my own rancher-/docker-compose.yml. I have deleted those.

rawmind0 commented:
@fhossfel , thanks for the changes....

LGTM...
